tests/components/glances/__init__.py

```diff
@@ -207,4 +207,13 @@
             "config": "UU",
         },
     },
+    "gpu": {
+        "GPU_0__NVIDIA GeForce RTX 3080": {
+            "name": "NVIDIA GeForce RTX 3080",
+            "temperature": 51,
+            "mem": 8.41064453125,
+            "proc": 26,
+            "fan_speed": 0,
+        }
+    },
```

Comment:
this is what I get from the glancesapi library. Shall I put in some string manipulation to clean that up or request changes upstream?